recipes/smallgenomeutilities/0.2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "smallgenomeutilities" %}
+{% set version = "0.2" %}
+{% set md5 = "f8b222892e9f18bb135efd5802a878a4" %}
+{% set sha256 = "802ca5f85a14e8243aa44e5f4fb771691f0c80689cf3ecb1edce5382ce0ecda3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/f8/e2/54b075294a3247595abf9f7d7314eddc04c77a049398157514b65087fe6a/{{ name|lower }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # not 'noarch python', because of skip version, and because of command line utils
```

Comment:
FYI: noarch packages can work with skipping version in the build section, but bioconda does not yet have support for this

Reply:
NOTE: was referring to [this](https://bioconda.github.io/linting.html#should-not-be-noarch) linting documatation entry.

Their logic it that version skipping and command-line utilities use specific versions of python and do not work across all python version. Thus they are not "cross platform" because they rely on a specific python.

I find this distinction weird, coming from an RPM background were *noarch* simply means *no binary stuff specific to one CPU* arch*itecture"